wiki/www/drafts.py

```diff
@@ -57,16 +49,6 @@ def get_context(context):
 @frappe.whitelist()
 def get_drafts(limit):
 	context = frappe._dict()
-	context.no_cache = 1
-	context.no_sidebar = 1
```

Comment:
no clue why this was added - isn't even used on the client side 🤔 

https://github.com/frappe/wiki/blob/8f324caf522db16536c83a93ee70b5a0cdd8bceb/wiki/www/drafts.html#L78-L95

Reply:
remove them👍🏻